src/utils/schema.ts

```diff
@@ -0,0 +1,11 @@
+import type {ESysViewType} from '../types/api/schema/sysView';
+
+export function prepareSystemViewType(type?: ESysViewType) {
+    if (!type) {
+        return undefined;
+    }
+    if (type.startsWith('E')) {
```

Comment:
That looks fragile - would be great if you left some comment what and why is 'E'

Reply:
Added comment